.travis.yml

```diff
@@ -13,7 +13,7 @@ install:
 - pip install coveralls==1.3.0
 - pip list
 script:
-- py.test --pep8 -m pep8
+- if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then pip install black; black --config rasa_core_sdk.toml --check .; fi
```

Comment:
while we are at it let's move it to a separate job stage and put it under `jobs include`:
```
- stage: test
   name: Check code style
   python: 3.6
   script: pip install black; black --config rasa_core_sdk.toml --check . 
```
(no need for the if there, should only be run once)